RecoMuon/MuonIdentification/plugins/ME0SegmentMatcher.h

```diff
@@ -0,0 +1,77 @@
+#ifndef ME0Segment_ME0SegmentMatcher_h
```

Comment:
similar to other producer headers, contents of this can be moved to .cc and .h removed